test/amm-wusdm/arbitrum/ProvideWusdmArbitrum.t.sol

```diff
@@ -0,0 +1,333 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./WusdmTestForkCommonArbitrum.sol";
+import "../../../contracts/libraries/errors/AmmErrors.sol";
+
+contract ProvideWusdmArbitrumTest is WusdmTestForkCommonArbitrum {
+
+    address userOne;
+
+    function setUp() public {
+        _init();
+        userOne = _getUserAddress(22);
+        _setupUser(userOne, 100_000 * 1e18);
+    }
+
+    function testtt() public {
+        assertTrue(true);
+    }
+
+    function testShouldExchangeRateBe1WhenNoProvideStEth() external {
+        //given
+
+        //when
+        uint exchangeRate = IAmmPoolsLensWusdm(IporProtocolRouterProxy).getIpWusdmExchangeRate();
```

Comment:
IAmmPoolsLensWusdm(IporProtocolRouterProxy) is given

Reply:
this test check init value